packages/nextjs/lib/utils.ts

```diff
@@ -1,17 +1,18 @@
 import fs from "fs"
+import path from "path"
 
 const isTest = process.env.NODE_ENV === "test"
 
-export function getPackageRoot(path: string): string {
-    let ret = path
+export function getPackageRoot(filePath: string): string {
+    let ret = filePath
     if (isTest) return ""
 
     while (ret != "") {
         try {
             fs.accessSync(ret)
             return ret
         } catch {
-            ret = ret.substring(0, ret.lastIndexOf("/"))
+            ret = path.dirname(ret)
         }
     }
```

Comment:
This is missing the actual `package.json` check (use `path.join()` to build the path)